packages/sidenav/src/sidenav.ts

```diff
@@ -45,6 +130,11 @@ export class SideNav extends LitElement {
             </nav>
         `;
     }
+
+    protected firstUpdated(changes: PropertyValues): void {
+        super.firstUpdated(changes);
+        this.tabIndex = 0;
```

Comment:
I didn't make this element an extension of `Focusable` at this time. It sort of is, but sort of isn't, and think reviewing this decision later makes sense.